src/communication/network_cl.c

```diff
@@ -892,8 +892,6 @@ net_client_request_no_reply (int request, char *argbuf, int argsize)
 
   error = NO_ERROR;
 
-  assert (request == NET_SERVER_LOG_SET_INTERRUPT);
-
```

Comment:
I want to keep the safe guard. You may also add a comment. 
To use this interface with a general request may cause races.

Reply:
done